app/controllers/api/foodbags_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::FoodbagsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    foodbag = current_user.foodbags.create(foodbag_params)
+    if foodbag.persisted?
+      render json: { message: 'Your foodbag was successfully created!' }, status: 201
+    else
+      render json: { message: foodbag.errors.full_messages.to_sentence }, status: 422
+    end
+  end
+
+  private
+
+  def foodbag_params
+    params.require(:foodbag).permit(:pickuptime, :status, :donor_id)
```

Comment:
why are you sending in the donor_id as a param still when we set it by creating the foodbag on that specific user?

Reply:
ok, will remove it.